mars/services/task/supervisor/processor.py

```diff
@@ -241,13 +240,14 @@ async def get_next_stage_processor(self) \
             self._preprocessor.done = True
             return
 
+        if not self._band_slots:
```

Comment:
Will it make `self._band_slots` never get updated after it first fetched bands from cluster service?